llvm/test/CodeGen/AMDGPU/check-subtarget-features.ll

```diff
@@ -1,5 +1,3 @@
-; RUN: not llc -global-isel=0 -mtriple=amdgcn -mcpu=gfx1100 -mattr=-wavefrontsize32,-wavefrontsize64 < %s 2>&1 | FileCheck %s -check-prefix=ERR -implicit-check-not=error:
```

Comment:
This diagnostics is now missing, BE will just initialize wavesize to default. If we want to catch this case we would still need to scan the original string.